pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -599,6 +599,11 @@ protected PartitionedTopicMetadata internalGetPartitionedMetadata(boolean author
     protected void internalDeletePartitionedTopic(AsyncResponse asyncResponse, boolean authoritative, boolean force) {
         try {
             validateWriteOperationOnTopic(authoritative);
+        } catch (WebApplicationException wae) {
+            log.debug("[{}] Failed to delete partitioned topic {}, redirecting to other brokers.",
```

Comment:
Please add check for the debug is enabled
if(log.isDebugEnable) {
  log.debug("....")
}

Reply:
Done.